src/libNode/Node.cpp

```diff
@@ -1264,8 +1264,13 @@ bool Node::ProcessTxnPacketFromLookup([[gnu::unused]] const bytes& message,
     }
   }
 
-  if (m_mediator.m_lookup->IsLookupNode(from) &&
-      from.GetPrintableIPAddress() != "127.0.0.1") {
+  if ((m_mediator.m_lookup->IsLookupNode(from) &&
```

Comment:
Perhaps we can remove `IsLookupNode` now here since we already do a check for `Lookup::VerifySenderNode` a few lines earlier.

Reply:
The verifier only check the txn origin’s PubKey. But the from Peer will change in each hop of gossip. So the earlier check can’t take the place of the IsLoopup check